flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -126,7 +126,7 @@
 	private boolean autoTypeRegistrationEnabled = true;
 
 	private boolean forceAvro = false;
-	private long autoWatermarkInterval = 0;
+	private long autoWatermarkInterval = 200;
```

Comment:
nit: Make the `200` a static var with a descriptive name instead of a "magic" number.

Reply:
I can add a `static final DEFAULT_WATERMARK_INTERVAL` but then it would basically be

```
private static final long DEFAULT_WATERMARK_INTERVAL = 200;
private long autoWatermarkInterval = DEFAULT_WATERMARK_INTERVAL;
```

except that we might but the default at the top. If you think it would help I can do it.

Reply:
I do not think it contributes to anything for now. It is just that if in the future someone, for whatever reason, wants to do a check like: `if (interval == DEFAULT) ...`, or he/she wants to reset the interval to the default, the literal `200` would be a bit obscure instead of a `DEFAULT_WM_INTERVAL`. But again, it does not add any value for now.